src/documentation/reviewing-doc-changes.md

```diff
@@ -0,0 +1,19 @@
+# Reviewing doc changes
+
+Most of the time, it is mostly reviewing that the documentation isn't wrong
+in any way and that it follows the
+[how to write documentation](./how-to-write-documentation.md) guideline.
+
+There is however something where we need to be extra careful: stability
+guarantees.
+
+## Stability guarantees
+
+First, short explanation about what a stability guarantee is: a statement in
+the document which explains what the item is doing in a precise case. For
```

Comment:
We should also mention that this can be circumvented by adding a "# Current Implementation" section and link to an example of one from the codebase

Reply:
Do you have a "rule" on when to use which by any chance?